pyEpiabm/pyEpiabm/sweep/update_place_sweep.py

```diff
@@ -129,19 +131,19 @@ def update_place_group(self, place, mean_capacity: float = 25,
                 person = random.choices(person_list, person_weights, k=1)[0]
             else:
                 i = random.randint(1, len(person_list))
-                person = person_list[i-1]
+                person = person_list[i - 1]
             # Checks person is not already in the place, and that they
             # haven't already been assigned to this place type.
             if ((person not in place.persons) and
                     (place.place_type not in person.place_types)):
-                if place.place_type == 5:
+                if place.place_type.name == "CareHome":
```

Comment:
ERROR 2: This error was found when we saw that there were no key workers or care home residents in the demographics output from the Gibraltar simulations. This came down to the above line with `place.place_type == 5`, which was incorrect syntax due to place_type being an enum.

Reply:
I see, yes this should have been picked up before but thank you very much for identifying it!